Makefile

```diff
@@ -42,6 +42,9 @@ GOLDFLAGS := -ldflags "-X $(PACKAGE_NAME)/pkg/util.AppGitState=${GIT_STATE} -X $
 	$(CMDS) go_test go_fmt e2e_test go_verify hack_verify hack_verify_pr \
 	$(DOCKER_BUILD_TARGETS) $(DOCKER_PUSH_TARGETS)
 
+# Docker build flags
+DOCKER_BUILD_FLAGS := --build-arg VCS_REF=$(GIT_COMMIT) $(DOCKER_BUILD_FLAGS)
```

Comment:
How come `$(DOCKER_BUILD_FLAGS)` is on the end of this var?

Reply:
From the command line. Something like:
```bash
DOCKER_BUILD_FLAGS="--build-arg http_proxy=http://proxy.company.com:3128" make buid
```